src/Goerli-DssSpell.t.sol

```diff
@@ -458,4 +508,347 @@ contract DssSpellTest is GoerliDssSpellTestBase {
+    RwaUrnLike           rwaurn_009          = RwaUrnLike(addr.addr("RWA009_A_URN"));
+    address              RWA009_CES_MULTISIG = addr.addr("RWA009_A_OUTPUT_CONDUIT");
+
+    function testRWA008_INTEGRATION_BUMP() public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        giveAuth(address(oracle), address(this));
+
+        (, address pip, , ) = oracle.ilks("RWA008-A");
+
+        assertEq(DSValueAbstract(pip).read(), bytes32(30_437_069 * WAD));
+
+        oracle.bump("RWA008-A", 40 * MILLION * WAD);
+
+        assertEq(DSValueAbstract(pip).read(), bytes32(40 * MILLION * WAD));
+    }
+
+    function testRWA008_INTEGRATION_TELL() public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        giveAuth(address(vat), address(this));
+        giveAuth(address(oracle), address(this));
+
+        (, , , uint48 tocPre) = oracle.ilks("RWA008-A");
+        assertTrue(tocPre == 0);
+        assertTrue(oracle.good("RWA008-A"));
+
+        vat.file("RWA008-A", "line", 0);
+        oracle.tell("RWA008-A");
+
+        (, , , uint48 tocPost) = oracle.ilks("RWA008-A");
+        assertTrue(tocPost > 0);
+        assertTrue(!oracle.good("RWA008-A"));
+    }
+
+    function testRWA008_INTEGRATION_TELL_CURE_GOOD() public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        giveAuth(address(vat), address(this));
+        giveAuth(address(oracle), address(this));
+
+        vat.file("RWA008-A", "line", 0);
+        oracle.tell("RWA008-A");
+
+        assertTrue(!oracle.good("RWA008-A"));
+
+        oracle.cure("RWA008-A");
+
+        assertTrue(oracle.good("RWA008-A"));
+        (, , , uint48 toc) = oracle.ilks("RWA008-A");
+        assertEq(uint256(toc), 0);
+    }
+
+    function testFailRWA008_INTEGRATION_CURE() public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        giveAuth(address(oracle), address(this));
+
+        oracle.cure("RWA008-A");
+    }
+
+    function testRWA008_INTEGRATION_TELL_CULL() public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        giveAuth(address(vat), address(this));
+        giveAuth(address(oracle), address(this));
+
+        assertTrue(oracle.good("RWA008-A"));
+
+        vat.file("RWA008-A", "line", 0);
+        oracle.tell("RWA008-A");
+
+        assertTrue(!oracle.good("RWA008-A"));
+
+        oracle.cull("RWA008-A", addr.addr("RWA008_A_URN"));
+
+        assertTrue(!oracle.good("RWA008-A"));
+        (, address pip, , ) = oracle.ilks("RWA008-A");
+        assertEq(DSValueAbstract(pip).read(), bytes32(0));
+    }
+
+    function testRWA008_OPERATOR_GET_RWA008_TOKEN() public {
+        assertEq(rwagem_008.balanceOf(rwaOperator_008), 1 * WAD);
+    }
```

Comment:
we should add this test for `RWA009` too.  Make sure the pause proxy has the gem.

Reply:
Done.

Reply:
Solving this.